CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 ## Changelog
 
+### Version 1.34
+Mar 10, 2020
+
+- Clean up deprecated code and unused imports ([PR-109](https://github.com/jenkinsci/durable-task-plugin/pull/109))
+- Upgrade parent pom to 3.54 ([PR-119](https://github.com/jenkinsci/durable-task-plugin/pull/119))
+- Explicitly close `ProcessPipeInputStream` to prevent agent-side OOM ([JENKINS-60960](https://issues.jenkins-ci.org/browse/JENKINS-60960))
```

Comment:
I try to clarify whether changes affect users by adding a prefix like `Fix:` or  `New Feature:` or `Internal:` or whatever, and then moving all of the `Internal:` stuff to the bottom. This looks like the only bugfix, so I'd move it to the top.

```suggestion
- Fix: Explicitly close `ProcessPipeInputStream` to prevent agent-side OOM ([JENKINS-60960](https://issues.jenkins-ci.org/browse/JENKINS-60960))
```